tests/snapshots/assets__check_file@packages-tidy-new-parser.typ-40.snap

```diff
@@ -443,7 +469,10 @@ snapshot_kind: text
             param-parser,
             curry.rest,
           )
-          if param-parser.state == "finished" {
+          if (
+            param-parser.state
+              == "finished"
+          ) {
```

Comment:
same

Reply:
one line is unable to contain this expr here